lib/kamal/configuration/proxy.rb

```diff
@@ -22,13 +22,13 @@ def ssl?
     proxy_config.fetch("ssl", false)
   end
 
-  def host
-    proxy_config["host"]
+  def hosts
+    proxy_config["hosts"] || proxy_config["host"]&.split(",") || []
   end
 
   def deploy_options
     {
-      host: proxy_config["host"],
+      host: hosts.present? ? hosts.join(",") : nil,
```

Comment:
We can just pass `hosts` here as kamal-proxy has support for `--host a.example.com --host b.example.com`

Reply:
Good point, thank you!

```shell
$ bin/kamal-proxy deploy nginx --target localhost:8080 --health-check-path / --host example.com --host anotherexample.com

$ bin/kamal-proxy list
Service  Host                            Target          State    TLS
nginx    example.com,anotherexample.com  localhost:8080  running  no
```

Updated the MR.